### ISSUE TITLE
Little problem with => when variables are used as key

### ISSUE THREAD

**Issue by fgimian** [rodjek/puppet-lint#472]
Hey guys, this code fails to lint:

```puppet
class { 'ssh':
  storeconfigs_enabled => false,
  server_options       => {
    "Match Host ${fqdn}" => {
      'PermitRootLogin' => 'without-password',
    }
  },
}
```

The error is:

```puppet
WARNING: indentation of => is not properly aligned on line 4
```

If I was to remove the double quotes and variable with the same alignment, it passes.

e.g. this code works

```puppet
class { 'ssh':
  storeconfigs_enabled => false,
  server_options       => {
    'Match Host --fqdn-' => {
      'PermitRootLogin' => 'without-password',
    }
  },
}
```

Any help would be greatly appreciated
Fotis

**Comment by bryanandrews** [comment]
I have this exact same problem. If I use a variable in a key I get a false indentation aligning warning. Here is my code example:

```
    class { '::ssh::server':
      storeconfigs_enabled => false,
      options              => {
        'Port'                            => ['22'],
        'PermitRootLogin'                 => 'without-password',
        'Subsystem'                       => 'sftp internal-sftp -l INFO',
        "Match User ${sftpuser}"          => {
          'ChrootDirectory'        => $sftpmountpath,
          'ForceCommand'           => 'internal-sftp -l INFO',
          'PasswordAuthentication' => 'yes',
          'AllowTcpForwarding'     => 'no',
          'X11Forwarding'          => 'no',
        },
      },
    }
```

**Comment by rnelson0** [comment]
Duplicate of #424.
